src/main/kotlin/com/autonomousapps/tasks/ProjectHealth.kt

```diff
@@ -22,9 +25,29 @@ abstract class ProjectHealth : DefaultTask() {
   @get:InputFile
   abstract val comprehensiveAdvice: RegularFileProperty
 
+  @get:Input
+  abstract val dependencyRenamingMap: MapProperty<String, String>
+
   @TaskAction fun action() {
+    val inputFile = comprehensiveAdvice.get().asFile
     val comprehensiveAdvice = comprehensiveAdvice.fromJson<ComprehensiveAdvice>()
 
+    val consoleReport = ConsoleReport.from(comprehensiveAdvice)
+    val advicePrinter = AdvicePrinter(consoleReport, dependencyRenamingMap.orNull)
+
+    val consoleText = advicePrinter.consoleText()
+    if (shouldNotBeSilent()) {
+      logger.quiet(consoleText)
+      if (consoleReport.isNotEmpty()) {
+        logger.quiet("See machine-readable report at ${inputFile.path}")
```

Comment:
What do you think of making this an `info` message?
I doubt this path is very important for regular task run, and I doubt this message is important for CI.

```suggestion
        logger.info("See machine-readable report at ${inputFile.path}")
```

Reply:
you may be right, but I'm hesitating right now because _I_ often use the path printed to ease debugging, and I don't want to have to deal with info-level logs.... Will think about it.